src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const object of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
use clear naming for variables